### PR TITLE
fix: #755 fix styles for responsive home

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

File: mw-webapp/src/logic/homePage/HomePage.module.scss [hunk 1]
```diff
@@ -84,17 +97,25 @@ $homeMaxWidthImage: 605px;
   font-size: 24px;
   gap: $homeGap;
   list-style-position: inside;
+
+  @media (max-width: $mediaWidthMobileBig) {
+    align-items: center;
+  }
 }
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/HomePage.module.scss:105]: it's a browser trick

File: mw-webapp/src/logic/homePage/HomePage.module.scss [hunk 2]
```diff
@@ -84,17 +97,25 @@ $homeMaxWidthImage: 605px;
...
 
-.aboutProjectGetStartedButton,
-.tryNowButton {
+.aboutProjectGetStartedButton {
   margin-right: auto;
+
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/HomePage.module.scss:108]: Done

File: mw-webapp/src/logic/homePage/HomePage.module.scss [hunk 3]
```diff
@@ -138,3 +162,11 @@ $homeMaxWidthImage: 605px;
   color: var(--secondaryTextColor);
   font-size: 24px;
 }
+
+// h2 {
+//   font-size: 48px;
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/HomePage.module.scss:167]: Deleted

File: mw-webapp/src/logic/homePage/HomePage.tsx [hunk 4]
```diff
@@ -72,13 +77,13 @@ export const HomePage = () => {
...
             text={LanguageService.home.aboutAppBlock.title[language]}
-            className={styles.goalsTitle}
+            classNameHeading={styles.goalsTitle}
           />
           <VerticalContainer className={styles.aboutProjectText}>
             <Title
               level={HeadingLevel.h3}
               text={LanguageService.home.aboutAppBlock.subTitle[language]}
-              className={styles.goalsTitle}
+              classNameHeading={styles.goalsSubTitle}
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/HomePage.tsx:86]: done

File: mw-webapp/src/logic/homePage/HomePage.tsx [hunk 5]
```diff
@@ -99,7 +104,9 @@ export const HomePage = () => {
             </div>
           </VerticalContainer>
           <Button
-            onClick={AuthService.logIn}
+            onClick={() => {
+              user ? navigate(pages.user.getPath({uuid: user.uuid})) : AuthService.logIn();
+            }}
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/HomePage.tsx:109]: done

File: mw-webapp/src/logic/homePage/HomePage.tsx [hunk 6]
```diff
@@ -150,7 +157,9 @@ export const HomePage = () => {
           </VerticalContainer>
           <HorizontalContainer className={styles.whatWeAreButtons}>
             <Button
-              onClick={AuthService.logIn}
+              onClick={() => {
+                user ? navigate(pages.user.getPath({uuid: user.uuid})) : AuthService.logIn();
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/HomePage.tsx:161]: Done

File: mw-webapp/src/logic/homePage/goalItem/GoalItem.module.scss [hunk 7]
```diff
@@ -2,13 +2,14 @@
 
 .goalItem {
   display: flex;
-  width: 317px;
+  max-width: 317px;
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/goalItem/GoalItem.module.scss:5]: Deleted

File: mw-webapp/src/logic/homePage/goalItem/GoalItem.module.scss [hunk 8]
```diff
@@ -2,13 +2,14 @@
...
   justify-content: left;
   gap: $gapMedium;
 }
 
 .goalTitle {
   color: var(--secondaryTextColor);
+  font-size: 24px;
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/goalItem/GoalItem.module.scss:12]: Done

File: mw-webapp/src/logic/homePage/HomePage.module.scss [hunk 9]
```diff
@@ -44,27 +46,38 @@ $homeMaxWidthImage: 605px;
...
+  }
 }
 
 .goalsTitle {
   color: var(--secondaryTextColor);
+  font-size: 48px;
+}
+
+.goalsSubTitle {
+  font-size: 24px;
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/HomePage.module.scss:73]: done

File: mw-webapp/src/logic/homePage/HomePage.module.scss [hunk 10]
```diff
@@ -44,27 +46,38 @@ $homeMaxWidthImage: 605px;
...
 
 .goals {
   flex-wrap: wrap;
+  justify-content: center;
   padding: $homePaddingSmallBlock $homePaddingBlock;
   border: 1px solid var(--homeBorderColor);
   border-radius: $homeBorderRadius;
   gap: $homeGapBig;
+
+  @media (max-width: $mediaWidthMobileBig) {
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/logic/homePage/HomePage.module.scss:62]: done

File: mw-webapp/src/component/title/Title.module.scss [hunk 11]
```diff
@@ -2,12 +2,16 @@
...
-  font-size: 48px;
+  @media (max-width: $mediaWidthMobileBig) {
+    font-size: 40px;
+  }
 }
 
-h3 {
-  font-size: 24px;
-}
+// h2 {
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/component/title/Title.module.scss:11]: deleted

File: mw-webapp/src/component/header/Header.module.scss [hunk 12]
```diff
@@ -39,6 +39,10 @@ $userNameFontSize: 16px;
 .rightBlock {
   align-items: center;
   gap: $gapBig;
+
+  @media (max-width: $mediaWidthMobileMedium) {
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/component/header/Header.module.scss:43]: done

File: mw-webapp/src/component/button/Button.module.scss [hunk 13]
```diff
@@ -139,4 +139,8 @@
   &:active {
     background-color: var(--homeActiveButtonColor);
   }
+
+  @media (max-width: $mediaWidthMobileBig) {
+    padding: 18px 24px;
+  }
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/component/button/Button.module.scss:145]: done

File: mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.module.scss [hunk 14]
```diff
@@ -11,4 +11,8 @@ $iconContainerSize: 20px;
   border: none;
   background: none;
   cursor: pointer;
+
+  @media (max-width: $mediaWidthMobileMedium) {
```
Comment (tritonJS826) [inline mw-webapp/src/logic/themeSwitcher/ThemeSwitcher.module.scss:15]: !

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**